db/bigtable.go

```diff
@@ -991,6 +991,37 @@ func (bigtable *Bigtable) GetValidatorSyncDutiesStatistics(validators []uint64,
 	return res, nil
 }
 
+func (bigtable *Bigtable) GetValidatorSyncCommitteesStats(validators []uint64, startEpoch uint64, endEpoch uint64) (types.SyncCommitteesStats, error) {
+	res, err := bigtable.getValidatorSyncDutiesHistory(validators, startEpoch, endEpoch)
+	if err != nil {
+		return types.SyncCommitteesStats{}, fmt.Errorf("error retrieving validator sync participations data from bigtable: %v", err)
+	}
+
+	retv := types.SyncCommitteesStats{}
+	for _, v := range res {
+		for _, r := range v {
+			slotTime := utils.SlotToTime(r.Slot)
+			if r.Status == 0 && time.Since(slotTime) <= time.Minute {
```

Comment:
This "one minute before now" is just an estimation for slots that appear to have been missed but are actually still scheduled or does this time come from somewhere?

Reply:
The source is the source code ... hence the name! :)

In all seriousness, this was the way it was treated before and I did not want to change the handling.